tensorflow/python/kernel_tests/linalg/sparse/csr_sparse_matrix_grad_test.py

```diff
@@ -84,6 +84,10 @@ def testLargeBatchSparseMatrixAddGrad(self):
     if not self._gpu_available:
       return
 
+    if test.is_built_with_rocm():
+      # sparse-matrix-add op is not yet supported on the ROCm platform
```

Comment:
This comment seems redundant. Same below.

Reply:
will push out a commit soon with all the redundant comments removed